cypress/e2e/shared/dashboardsView.test.ts

```diff
@@ -1392,4 +1392,268 @@ csv.from(csv: data) |> filter(fn: (r) => r.bucket == v.bucketsCSV)`
+            `test,container_name=cool dopeness=14 ${now - 1400}000000`,
+            `test,container_name=beans dopeness=10 ${now - 1600}000000`,
+          ],
+          'schmucket'
+        )
+      })
+      cy.getByTestID('button').click()
+      cy.getByTestID('switch-to-script-editor').should('be.visible')
+      cy.getByTestID('switch-to-script-editor').click()
+      cy.getByTestID('toolbar-tab').click()
+      const query1 = `from(bucket: "schmucket")
+  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+  |> filter(fn: (r) => r["container_name"] == "cool")`
+      cy.getByTestID('flux-editor')
+        .should('be.visible')
+        .click()
+        .focused()
+        .type(query1)
+      cy.getByTestID('overlay').within(() => {
+        cy.getByTestID('page-title').click()
+        cy.getByTestID('renamable-page-title--input')
+          .clear()
+          .type('blah')
+        cy.getByTestID('save-cell--button').click()
+      })
+    })
+
+    it('can enable the auto refresh process via the modal, then manually stop the process via the stop button', done => {
+      cy.getByTestID('enable-auto-refresh-button').click()
+      cy.getByTestID('auto-refresh-overlay').should('be.visible')
+      cy.getByTestID('auto-refresh-overlay').within(() => {
+        cy.getByTestID('autorefresh-dropdown--button').click()
+        cy.getByTestID('auto-refresh-5s').click()
+        cy.getByTestID('timerange-dropdown').click()
+      })
+      cy.getByTestID('timerange-popover--dialog').within(() => {
+        cy.getByTestID('timerange--input')
+          .clear()
+          .type(`${jumpAheadTime('00:20:00')}`)
+        cy.getByTestID('daterange--apply-btn').click()
+      })
+      cy.intercept('POST', '/query', req => {
+        req.alias = 'refreshQuery'
+      })
+      cy.getByTestID('auto-refresh-overlay').within(() => {
+        cy.getByTestID('refresh-form-activate-button').click()
+      })
+      cy.wait('@refreshQuery')
+      cy.wait('@refreshQuery')
+      cy.getByTestID('enable-auto-refresh-button').click()
+      cy.getByTestID('enable-auto-refresh-button').then(el => {
+        expect(el[0].getAttribute('title')).to.equal('Enable Auto Refresh')
+      })
+      cy.wait('@refreshQuery')
+      cy.on('fail', err => {
+        expect(err.message).to.equal(
+          'Timed out retrying after 5000ms: `cy.wait()` timed out waiting `5000ms` for the 3rd request to the route: `refreshQuery`. No request ever occurred.'
+        )
+        done()
+      })
+    })
+
+    it('can timeout on a preset timeout selected by the user', () => {
+      cy.getByTestID('enable-auto-refresh-button').click()
+      cy.getByTestID('auto-refresh-overlay').should('be.visible')
+      cy.getByTestID('auto-refresh-overlay').within(() => {
+        cy.getByTestID('autorefresh-dropdown--button').click()
+        cy.getByTestID('auto-refresh-5s').click()
+        cy.getByTestID('timerange-dropdown').click()
+      })
```

Comment:
this is all reused. If we're looking to set this as the basis for our tests, we should probably put this in the beforeEach

Reply:
great catch!